### PR TITLE
Colo changes

### PR DESCRIPTION
**Pull request by justinengland**

### CODE REVIEW

File: userify/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    mode: 0777
+  notify: start_userify
+
+- name: install userify
+  shell: './installer.sh'
+  args:
+    chdir: /tmp
+    creates: /opt/userify
+  environment:
+    api_id: "{{userify_api_id}}"
```
Comment (cmmarslender) [inline userify/tasks/main.yml:16]: If we're going to use variables, we should have defaults definted in `<role>/defaults/main.yml` (even if they're just empty strings or something) as documentation of the possible variables we can override

File: arm-runner/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+
+# userify/handlers/main.yml
+
+- name: start_userify
```
Comment (cmmarslender) [inline arm-runner/handlers/main.yml]: wrong service

File: arm-runner/handlers/main.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  systemd:
+    name: userify
```
Comment (cmmarslender) [inline arm-runner/handlers/main.yml]: wrong service

File: userify/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+# userify/tasks/main.yml
+- name: get userify shim
+  get_url:
+    url: https://shim.userify.com/installer.sh
+    dest: /tmp
+    mode: 0777
+  notify: start_userify
```
Comment (cmmarslender) [inline userify/tasks/main.yml:8]: We should probably only download this if /opt/userify doesn't exist?

File: arm-runner/meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: BSD-3-Clause
```
Comment (cmmarslender) [inline arm-runner/meta/main.yml:17]: Should this be apache-2.0 to match the blockchain stuff?

File: arm-runner/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+  # chia-runner/tasks/main.yml
+  - name: Clone runner code and unzip
+    shell: mkdir actions-runner && cd actions-runner && curl -O -L https://github.com/actions/runner/releases/download/v2.273.5/actions-runner-linux-x64-2.273.5.tar.gz && tar xzf ./actions-runner-linux-x64-2.273.5.tar.gz
```
Comment (cmmarslender) [inline arm-runner/tasks/main.yml]: Would be better to break this down into the actual ansible modules for creating directories, downloading files, etc, so that ansible can be smart about not downloading things when we already have the stuff installed

File: arm-runner/tasks/main.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+  - name: Copy Service File TO AMI
+    copy:
+      src: files/runner-service
+      dest: /etc/systemd/system/github-runner.service
+      owner: root
+      group: root
+      mode: 0755
+
+  - name: Extend Packer Max Timeout
```
Comment (cmmarslender) [inline arm-runner/tasks/main.yml:16]: Is this relevant to this runner install?

File: mac-runner/handlers/main.yml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
+
+# userify/handlers/main.yml
+
+- name: start_userify
```
Comment (cmmarslender) [inline mac-runner/handlers/main.yml]: Wrong service

File: mac-runner/meta/main.yml [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: BSD-3-Clause
```
Comment (cmmarslender) [inline mac-runner/meta/main.yml:17]: Apache 2.0 ?

File: mac-runner/tasks/main.yml [hunk 10]
```diff
@@ -0,0 +1,18 @@
+---
+# chia-runner/tasks/main.yml
+- name: Clone runner code and unzip
+  shell: mkdir actions-runner && cd actions-runner && curl -O -L https://github.com/actions/runner/releases/download/v2.273.5/actions-runner-linux-x64-2.273.5.tar.gz && tar xzf ./actions-runner-linux-x64-2.273.5.tar.gz
```
Comment (cmmarslender) [inline mac-runner/tasks/main.yml]: Same comment as above - would be nice to break this up into the ansible tasks that are designed for directories, donwloading files, etc

File: mac-runner/tasks/main.yml [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+- name: Copy Service File TO AMI
+  copy:
+    src: files/runner-service
+    dest: /etc/systemd/system/github-runner.service
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Extend Packer Max Timeout
+  shell: export AWS_POLL_DELAY_SECONDS=60 && export AWS_MAX_ATTEMPTS=180
```
Comment (cmmarslender) [inline mac-runner/tasks/main.yml]: Needed?

File: userify/meta/main.yml [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: BSD-3-Clause
```
Comment (cmmarslender) [inline userify/meta/main.yml:17]: Apache 2.0 ?

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: userify/tasks/main.yml (hunk 4 above)
Comment (justinengland) [reply userify/tasks/main.yml:8]: per our discussion. these modules are intentionally desctructive.
